src/modules/roles/roles.module.ts

```diff
@@ -0,0 +1,7 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { RoleEntity } from './entities/role.entity';
+import { UserRoleEntity } from './entities/user-role.entity';
+
+@Module({ imports: [TypeOrmModule.forFeature([RoleEntity, UserRoleEntity])] })
+export class RolesModule {}
```

Comment:
if we're not using service or any controller you can remove this module, just put the RoleEntity and UserRoleEntity in the AuthModule to use ir

Reply:
okay!